### PR TITLE
user api and index changed

### PR DESCRIPTION
**Pull request by Ehasaniceiu04**

### CODE REVIEW

File: gitpro-demo.api/Pages/Index.cshtml [hunk 1]
```diff
@@ -5,6 +5,6 @@
 }
 
 <div class="text-center">
-    <h1 class="display-4">Welcome</h1>
+    <h1 class="display-4">Welcome my website</h1>
```
Comment (ehasan16) [inline gitpro-demo.api/Pages/Index.cshtml:8]: letter w of website should be capitalize.

File: gitpro-demo.api/UserController.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace gitpro_demo.api
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class UserController : ControllerBase
```
Comment (ehasan16) [inline gitpro-demo.api/UserController.cs:8]: Rename user controller to AccountController

File: gitpro-demo.api/Pages/Index.cshtml (hunk 1 above)
Comment (Ehasaniceiu04) [reply gitpro-demo.api/Pages/Index.cshtml]: done

File: gitpro-demo.api/UserController.cs (hunk 2 above)
Comment (Ehasaniceiu04) [reply gitpro-demo.api/UserController.cs]: done

**Pull request merged**